client/comms/tls.go

```diff
@@ -0,0 +1,35 @@
+package comms
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"net/url"
+)
+
+// TLSConfig prepares a *tls.Config struct using the provided cert.
+func TLSConfig(URL string, cert []byte) (*tls.Config, error) {
+	if len(cert) == 0 {
+		return nil, nil
+	}
```

Comment:
https://github.com/decred/dcrdex/pull/2004/files#r1054897301

@martonp, you are correct and it is already handled.